BoxContentSDK/BoxContentSDK/OAuth2/BOXOAuth2Session.h

```diff
@@ -156,6 +156,12 @@
  */
 - (NSString *)redirectURIString;
 
+/**
+ * Returns the randomly generated nonce used to prevent spoofing attack during login
+ * @return generated nonce
+ */
+- (NSString *)nonce;
```

Comment:
As far as I can tell, we are not allowing users to modify this string by exposing it in the header. Also once generated, it remains the same so we wont run into any inconsistency in value.